source/reference/versioning.txt

```diff
@@ -39,35 +39,26 @@ asked to upgrade in order to receive support.
 MongoDB Server Version Compatibility and Support
 ------------------------------------------------
 
-{+c2c-product-name+} supports the following versions of MongoDB Server:
+.. include:: /includes/fact-minimum-server-version-support.rst
 
+``mongosync`` doesn't support MongoDB rapid releases, such as 6.1 or 6.2.
 
-.. list-table::
-   :header-rows: 1
-   :stub-columns: 1
+.. include:: /includes/fact-minimum-versions.rst
 
-   * - ``mongosync`` Version
-     - MongoDB Server Version
+.. include:: /includes/cross-version-sync.rst
 
-   * - 1.6.0
-     - 7.0.x
 
-   * - 1.0.0
-     - 6.0.x
-
-Considerations
---------------
+Version-Related Considerations
+------------------------------
 
 {+c2c-product-name+} has the following version related considerations:
 
 Same Server Version
```

Comment:
Looking at the docs now, I actually think it might be less confusing if we just removed the "Same Server Version" section since we already reference the info here in the above section

Reply:
Noted and updated!